content/docs/setup/kubernetes/install/multicluster/gateways/index.md

```diff
@@ -1,24 +1,28 @@
 ---
-title: Gateway Connectivity
-description: Install an Istio mesh across multiple Kubernetes clusters using Istio Gateway to reach remote pods.
+title: Multiple control planes
+description: Install an Istio mesh across multiple Kubernetes clusters with locally deployed control planes.
 weight: 2
 aliases:
-    - /docs/setup/kubernetes/multicluster-install/gateways/
+  - /docs/setup/kubernetes/multicluster-install/gateways/
+  - /docs/examples/multicluster/gateways/
+  - /docs/tasks/multicluster/gateways/
 keywords: [kubernetes,multicluster,federation,gateway]
 ---
 
-Follow this flow to install an Istio [multicluster service mesh](/docs/concepts/multicluster-deployments/)
-when services in a Kubernetes cluster use gateways to communicate with services in other clusters.
+Follow this guide to install an Istio
+[multicluster service mesh](/docs/concepts/multicluster-deployments/#multicluster-service-mesh)
+with locally deployed Istio control planes in every cluster and using gateways to
```

Comment:
locally/individually

Reply:
@frankbu re the wording changes, I think as you suggested we should leave the release notes as is.  Can you address this non-release notes change?  The wording of individually is a little misleading.